crates/ruff_python_ast/src/nodes.rs

```diff
@@ -495,14 +495,14 @@ impl Ranged for DictItem {
     }
 }
 
-/// See also [Dict](https://docs.python.org/3/library/ast.html#ast.Dict)
-#[derive(Clone, Debug, PartialEq)]
-pub struct ExprDict {
-    pub range: TextRange,
-    pub items: Vec<DictItem>,
-}
+// /// See also [Dict](https://docs.python.org/3/library/ast.html#ast.Dict)
+// #[derive(Clone, Debug, PartialEq)]
+// pub struct ExprDict {
+//     pub range: TextRange,
+//     pub items: Vec<DictItem>,
+// }
 
-impl ExprDict {
+impl crate::ExprDict {
```

Comment:
In other places, we are using

```rust
use crate::{self as ast};
```

and then `ast::ExprDict` instead of `crate::ExprDict`.  That makes it mimic how this crate is used in the rest of the ruff codebase.